src/frontend/Sider/Behandling/Inngangsvilkår/PassBarn/PassBarn.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+
+import { Vilkårsregler } from '../../../../typer/regel';
+import { Inngangsvilkårtype, Vilkårsresultat, Vilkårsvurdering } from '../../vilkår';
+import { Vilkårpanel } from '../../Vilkårspanel/Vilkårpanel';
+import { VilkårpanelInnhold } from '../../Vilkårspanel/VilkårpanelInnhold';
+import VisEllerEndreVurdering from '../../Vilkårvurdering/VisEllerEndreVurdering';
+
+interface Props {
+    vilkårsregler: Vilkårsregler<Inngangsvilkårtype.PASSBARN>;
+    vilkårsvurdering: Vilkårsvurdering;
+}
+
+const PassBarn: React.FC<Props> = ({ vilkårsregler, vilkårsvurdering }) => {
+    const vilkårsett = vilkårsvurdering.vilkårsett.filter(
+        (v) => v.vilkårType === Inngangsvilkårtype.PASSBARN
+    );
+
+    if (vilkårsett.length === 0) {
+        return <div>Mangler vurderinger for pass av barn</div>;
+    }
```

Comment:
Er denne nødvendig? Eller vil man i map senere bare ikke få noe resultat?

Reply:
Er dette en feilsituasjon som ikke skal skje?

Reply:
Akkurat denne kan vel kanskje skje, iaf i test nå da
Senere kan den skje hvis vi vises PassBarn for andre stønader enn TilsynBarn 😬

Reply:
Eller at man ikke har noen barn da.. ?

Reply:
Tja vil skje for dere nå fordi dere ikke har fått vilkårne. De opprettes første gang man er inne og oppdateres ikke senere. For alle nye behandlinger lages de. Det kan jo seff skje en feil. 

Det jeg lurer på er mest om hvis lista er tom, vil den da mappe gjennom noe? Eller vil map bare returnere ikke noe fordi lista er tom? For da er i såfall denne sjekken unød

Reply:
Men er kanskje bedre at det vises en feilmelding enn ikke noe når jeg tenker meg om. Den kræsjer hvertfall ikke uten denne. Da er det bare at man ikke vet at det skulle vært flere vilkår der